components/molecules/charts/schedule/helper/columns.js

```diff
@@ -0,0 +1,153 @@
+/**
+ * Column
+ */
+
+// React
+import React from 'react'
+import styled from 'styled-components'
+import {
+  endOfMonth,
+  endOfDay,
+  endOfWeek,
+  startOfMonth,
+  startOfWeek,
+  startOfDay,
+  setMonth,
+  setDay
+} from 'date-fns'
+// IUI
+import ColumnPagination from './columnPagination'
+import Tooltip from '../../../../atoms/tooltip/tooltip'
+
+const DATE_TYPE = {
+  DAY: 'day',
+  MONTH: 'month',
+  YEAR: 'year',
+  WEEK: 'week'
+}
+
+const columnPattern = {
+  day: [...Array(12).keys()],
+  month: ['Week1', 'Week2', 'Week3', 'Week4'],
+  year: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'],
+  week: ['Sat', 'Sun', 'Mon', 'Tue', 'Wed', 'Thu', 'Fri']
+}
+
+const formatCell = (handleClick, month, row, otherData) => {
+  return row[month] && row[month][0] ? (
+    row[month][2] ? (
+      <Tooltip content={row[month][2]}>
+        <Wrapper
```

Comment:
@Ojaghi1991 Wrapper is duped below - keep DRY and sepearate into file and component of itself